setup.py

```diff
@@ -1,6 +1,5 @@
 import setuptools
 from distutils.util import convert_path
-from setuptools.command.install import install
```

Comment:
what is the reason for this? just wondering

Reply:
added